src/ArrayUtils.php

```diff
@@ -198,7 +198,7 @@ public static function isHashTable(mixed $value, $allowEmpty = false)
      */
     public static function inArray(mixed $needle, array $haystack, $strict = false)
     {
-        if (! $strict) {
+        if ($strict === false) {
```

Comment:
Careful with this change: `(bool)` would be better, as `0` was behaving this way before, and now doesn't

Reply:
👍 - I'll add the cast. I think we should also deprecate this method and remove in 4.x - I don't see what it achieves beyond `in_array`, and there are also *no tests* for this method